### PR TITLE
Update fr translations for 2.0.9.9000

### PR DESCRIPTION
**Pull request by jplecavalier**
For #2659.

@maelle or any other French speaking user of the package, please feel free to review and suggest edits. I would particularly like to know what you think about "superseded" and "experimental", should we write "remplacée" and "expérimentale" with their feminine style since it will most likely reference to a function (feminine word in French)?

@Hadley I did not compile the changes, I thought you would do it when the issue will be completely resolved.

### CODE REVIEW

File: po/R-fr.po [hunk 1]
```diff
@@ -171,7 +171,7 @@ msgstr "Version CRAN : %s"
 
 #: build-reference-index.R:22
 msgid "Package index"
-msgstr ""
+msgstr "Indice du paquetage"
```
Comment (maelle) [inline po/R-fr.po:174]: je crois qu'on dit en fait "index" ? https://www.linguee.fr/francais-anglais/search?source=auto&query=index "Indice" m'a surprise. :thinking:

File: po/R-fr.po [hunk 2]
```diff
@@ -237,19 +235,19 @@ msgstr "Plus d'articles..."
 
 #: package.R:300
 msgid "deprecated"
-msgstr ""
+msgstr "obsolète"
 
 #: package.R:301
 msgid "superseded"
-msgstr ""
+msgstr "remplacé"
```
Comment (maelle) [inline po/R-fr.po:242]: yep I guess the feminine one would make more sense at the moment

File: po/R-fr.po [hunk 3]
```diff
@@ -237,19 +235,19 @@ msgstr "Plus d'articles..."
...
 
 #: package.R:302
 msgid "experimental"
-msgstr ""
+msgstr "expérimental"
 
 #: package.R:303
 msgid "stable"
-msgstr ""
+msgstr "stable"
```
Comment (maelle) [inline po/R-fr.po:250]: stable et obsolète sont les meilleurs, aucun souci de forme :ok_hand:

**Review by maelle — COMMENTED** [review]
Merci !! :pray:

File: po/R-fr.po (hunk 1 above)
Comment (jplecavalier) [reply po/R-fr.po:174]: On y va avec index sans problème!

**Pull request merged**